security/rbac/README.md

```diff
@@ -133,6 +133,33 @@ confluent iam rolebinding create --principal User:$CLIENT --role ResourceOwner -
 # REST Proxy Admin: no additional administrative rolebindings required because REST Proxy just does impersonation
 
 # Producer/Consumer
- confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Topic $TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
- confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Group $CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Topic $TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Group $CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
+```
+
+
+## KSQL
+
+* [delta_configs/ksql-server.properties.delta](delta_configs/ksql-server.properties.delta)
+* Role bindings:
+
+```bash
+# KSQL Server Admin
+confluent iam rolebinding create --principal User $ADMIN_KSQL --role ResourceOwner --resource Topic _confluent-ksql-${KSQL_SERVICE_ID}_command_topic --kafka-cluster-id $KAFKA_CLUSTER_ID
```

Comment:
Nitpicking: would be a bit more readable with some line breaks, the commands are long and there are a lot of them, easy to get lost. How about
```
confluent iam rolebinding create \
    --principal User:$ADMIN_KSQL \
    --resource Topic:_confluent-ksql-${KSQL_SERVICE_ID}_command_topic \
    --kafka-cluster-id $KAFKA_CLUSTER_ID
```

Reply:
See https://github.com/confluentinc/examples/pull/204#discussion_r304157477

Reply:
@stan-confluent thanks for the suggestion. I filed a separate jira DEVX-959 to track changing it to multi-line for all services, not just KSQL